.github/workflows/common-flow-host.yml

```diff
@@ -81,3 +112,13 @@ jobs:
           name: Sanitizer Logs
           path: tests/pytests/logs/*.log.*
           if-no-files-found: ignore
+
+      - name: Fail flow if tests failed
+        if: ${{ steps.test1.outcome == 'failure' || steps.test2.outcome == 'failure' || steps.test3.outcome == 'failure' || steps.test4.outcome == 'failure' }}
+        run: exit 1
+      # - name: Fail flow if sanitizer failed
+      #   if: ${{ inputs.san }}
+      #   run: |
+      #     if [[ -n "$(ls tests/pytests/logs/*.log.* 2>/dev/null)" ]]; then
+      #       exit 1
+      #     fi
```

Comment:
Because currently there are some logs generated. I made another PR that fixes what causes these logs, until it is merged we don’t want to fail the flow (but we probably will in the future)

Reply:
[This PR](https://github.com/RediSearch/RediSearch/pull/3784) will fix the warning, and then we could uncomment this verification

Reply:
I see, so let's merge it and uncomment